### PR TITLE
0.9.0-rc -> release

### PR DESCRIPTION
**Pull request by joshuahuburn**
#### Description
Merges the 0.9.0-rc into release (with some changelog fix-ups)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -9,6 +9,111 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased-`x.y.z`] - 2020-xx-xx
```
Comment (mironec) [inline CHANGELOG.md:10]: 0.9.0?

File: CHANGELOG.md [hunk 2]
```diff
@@ -9,6 +9,111 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
 
+### New Known Issues:
+- After upgrading to Unreal Engine `4.24.3` using `git pull`, you may be left in a state where several `.isph` and `.ispc` files are missing. This state produces [compile errors](https://forums.unrealengine.com/unreal-engine/announcements-and-releases/1695917-unreal-engine-4-24-released?p=1715142#post1715142) when you build the engine. You can fix this by running `git restore .` in the root of your `UnrealEngine` repository.
+
+### Breaking Changes:
+- Simulated Player worker configurations now require a dev auth token and deployment flag instead of a login token and player identity token. See the Example Project for an example of how to set this up.
+
+### Features:
+- Unreal Engine `4.24.3` is now supported. You can find the `4.24.3` version of our engine fork [here](https://github.com/improbableio/UnrealEngine/tree/4.24-SpatialOSUnrealGDK-preview).
```
Comment (mironec) [inline CHANGELOG.md:19]: maybe this should now point to release?

File: CHANGELOG.md [hunk 3]
```diff
@@ -9,6 +9,111 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
+- Switched off default rpc-packing. This can still be re-enabled in SpatialGDKSettings.ini
+- Starting a local deployment now checks if the required runtime port is blocked and allows the user to kill it
+- A configurable actor component 'SpatialPingComponent' is now available for player controllers to measure round-trip ping to their current authoritative server worker. The latest ping value can be accessed raw through the component via 'GetPing()' or otherwise via the rolling average stored in 'PlayerState'.
+- The `GenerateSchema`, `GenerateSchemaAndSnapshots`, and `CookAndGenerateSchema` commandlets can be invoked with the `-AdditionalSchemaCompilerArguments="..."` command line switch to output additional compiled schema formats. If no such switch is provided, only the schema descriptor will be produced. This switch's value should be a subset of the arguments that can be passed to the schema compiler directly (e.g., `--bundle_out="path/to/bundle.sb"`). A full list of possibles values is available via the [schema compiler documentation](https://docs.improbable.io/reference/14.2/shared/schema/introduction#schema-compiler-cli-reference)
```
Comment (mironec) [inline CHANGELOG.md:31]: Points to 14.2 version of docs, maybe update to 14.5? or latest?

File: SpatialGDK/Extras/internal-documentation/release-process.md [hunk 4]
```diff
@@ -174,21 +144,17 @@ Copy the latest release notes from `CHANGELOG.md` and paste them into the releas
 1. In `UnrealGDK`, merge `release` into `master`.
 
 **Documentation**
-1. Publish the docs to live using Improbadoc commands listed [here](https://improbableio.atlassian.net/wiki/spaces/GBU/pages/327485360/Publishing+GDK+Docs).
-1. Update the [roadmap](https://github.com/spatialos/UnrealGDK/projects/1), moving the release from **Planned** to **Released**, and linking to the release.
```
Comment (mironec) [inline SpatialGDK/Extras/internal-documentation/release-process.md:178]: Any reason this was deleted along with the line below?

File: ci/unreal-engine.version [hunk 5]
```diff
@@ -1,2 +1,2 @@
-UnrealEngine-21c4d81d3bc5db3eab66070d9b6f6c905c60299f
-UnrealEngine-f6a7b93f227ddc5e9fa9aa84d0dd16ac341f36ee
+HEAD 4.24-SpatialOSUnrealGDK
```
Comment (mironec) [inline ci/unreal-engine.version:1]: This will need to be updated to a pinned commit

File: SpatialGDK/SpatialGDK.uplugin [hunk 6]
```diff
@@ -11,15 +11,15 @@
   "MarketplaceURL": "",
```
Comment (mironec) [inline SpatialGDK/SpatialGDK.uplugin:11]: Can't comment properly, but this needs an update to 0.9 in the lines above.
If this is missing from the release process, it needs to be added there.

Comment (m-samiec) [reply SpatialGDK/SpatialGDK.uplugin:11]: Yeah VersionName and Version both need updating

File: SpatialGDK/Extras/internal-documentation/release-process.md (hunk 4 above)
Comment (joshuahuburn) [reply SpatialGDK/Extras/internal-documentation/release-process.md:178]: We don't use these docs anymore

Comment (mironec) [reply SpatialGDK/Extras/internal-documentation/release-process.md:178]: Really? I thought Ernie kept the known issues on GitHub updated

**Comment by joshuahuburn** [comment]
Dropping in favour of https://github.com/spatialos/UnrealGDK/pull/2091

**Pull request closed without merging**